mobile/web/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
Not sure if this is bloat, but remove any unused files or generally irrelevant web app files since this is focused on mobile app dev.

Reply:
unused it autogenerated a flutter for web section. I decided to keep it for testing purposes